content/platform/swarm-learning/home.md

```diff
@@ -0,0 +1,66 @@
+---
+title: HPE Swarm Learning
+version: 1.2.0
+description: "HPE Swarm Learning is a decentralized, privacy-preserving Machine
+  Learning framework. "
+image: /img/platforms/guid-e80d248e-e754-498e-99d6-67508092f779-high.jpg
+width: large
+priority: 0
+frontpage: false
+active: false
+---
+HPE Swarm Learning is a decentralized, privacy-preserving Machine Learning framework. This framework utilizes the computing power at, or near, the distributed data sources to run the Machine Learning algorithms that train the models. It uses the security of a blockchain platform to share learnings with peers in a safe and secure manner. In  HPE Swarm Learning, training of the model occurs at the edge, where data is most recent, and where prompt, data-driven decisions are mostly necessary. In this completely decentralized architecture, only the insights learned are shared with the collaborating ML peers, not the raw data. This tremendously enhances data security and privacy.
+
+- - -
+
+## Articles
+
+* **[2﻿021 Nature Paper - Swarm Learning for decentralized and confidential clinical machine learning](https://www.nature.com/articles/s41586-021-03583-3)**
+* **[HPE Labs - Swarm Learning: Turn your distributed data into competitive edge](https://www.hpe.com/psnow/doc/a50000344enw?jumpid=in_lit-psnow-red)**
+* **[Research gate - Swarm Learning as a privacy-preserving machine learning approach for disease classification](https://www.researchgate.net/publication/342495847_Swarm_Learning_as_a_privacy-preserving_machine_learning_approach_for_disease_classification)**
+
+- - -
+
+## Learn more about HPE Swarm Learning and its underlying technology
+
+**\-﻿ What's All the Buzz about Swarm Learning?**
+
+In this video, Dr. Eng Lim Goh highlights the challenges of data privacy in the organizations to use siloed data to be used to bring out collective insights. Also, he explains HPE Swarm learning has the potential to change our lives and solve some of the world's most pressing and complex problems.
```

Comment:
recommend rewriting the first sentence to:
"In this video, Dr. Eng Lim Goh highlights the challenges of data privacy for organizations using siloed data to bring about collective insights."

In the second sentence, since you've been stating it as "Swarm Learning" (indicating that it's essentially a product or program name), then the "L" should be capitalized in this case as well.

Reply:
done